spec/rubocop/cop/rspec/repeated_example_group_body_spec.rb

```diff
@@ -0,0 +1,216 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RepeatedExampleGroupBody do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated describe body' do
+    expect_offense(<<-RUBY)
+      describe 'doing x' do
+      ^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [5]
```

Comment:
You are using `line(s)` in the message. There is no test of multiple repetitions though.